### ISSUE TITLE
ChromeDriver not working

### ISSUE THREAD

**Issue by andrcuns** [butomo1989/docker-android#2]
Hi,
Great project, just what I was looking for to support our mobile testing with emulator.
One issue is that libgconf-2.so.4 is missing for chromedriver to work. Please consider adding libgconf-2-4 to installed packages for it to work.

**Comment by butomo1989** [comment]
Hi @andrcuns,

Chrome driver are available only for android version 7.0 and 7.1.1. please try with one of those versions and let me know if you still need anything else. 
btw, I have a simple UI test that is written in python for mobile website testing with chrome browser: https://github.com/butomo1989/docker-android/blob/master/example/android/python/msite_simple_chrome.py

**Comment by andrcuns** [comment]
I was trying it out with 7.1.1, selenium version 3.4.0 and java and chromedriver fails with libgconf-2.so.4 missing. I rebuilt the image locally with added libgconf-2-4 library and it works fine afterwards.

**Comment by butomo1989** [comment]
thank you @andrcuns. it will be great if you can post the log and steps how you reproduce the problem here.

**Comment by andrcuns** [comment]
Maybe I am missing something but had to actually fight a bit to copy the output of Appium server since doesn't look like VNC shares the clipboard and couldn't find where it is stored in some log files :D

Images tested:
butomo1989/docker-android-x86-7.1.1
butomo1989/docker-android-x86-6.0
most likely other as well.

Following appium client is used:
io.appium:java-client:4.1.2

Following capabilities are set:
```groovy
DesiredCapabilities capabilities = new DesiredCapabilities()
capabilities.setCapability(DEVICE_NAME, "Android Emulator")
capabilities.setCapability(PLATFORM_NAME, "Android")
capabilities.setCapability(BROWSER_NAME, MobileBrowserType.CHROME)
capabilities.setCapability(NEW_COMMAND_TIMEOUT, 15)
capabilities.setCapability("clearSystemFiles",true)
```
Following scenario is executed:
Given user is on some web page

Following exception is thrown:
```
[debug] [Chromedriver] Changed state to 'starting'
[AndroidBootstrap] [BOOTSTRAP LOG] [debug] Client connected
[Chromedriver] Set chromedriver binary as: /usr/lib/node_modules/appium/node_modules/appium-chromedriver/chromedriver/linux/chromedriver_64
[Chromedriver] Killing any old chromedrivers, running: pkill -15 -f "/usr/lib/node_modules/appium/node_modules/appium-chromedriver/chromedriver/linux/chromedriver_64.*--port=8000"
[Chromedriver] No old chromedrivers seemed to exist
[Chromedriver] Spawning chromedriver with: /usr/lib/node_modules/appium/node_modules/appium-chromedriver/chromedriver/linux/chromedriver_64 --url-base=wd/hub --port=8000 --adb-port=5037
[Chromedriver] [STDERR] /usr/lib/node_modules/appium/node_modules/appium-chromedriver/chromedriver/linux/chromedriver_64: error while loading shared libraries: libgconf-2.so.4: cannot open shared object file: No such file or directory
[Chromedriver] Chromedriver exited unexpectedly with code 127, signal null
[debug] [Chromedriver] Changed state to 'stopped'
```

Proposed solution:
Installing libgconf-2-4 seems to fix the problem.

**Comment by butomo1989** [comment]
thank you @andrcuns. I will install libgconf-2-4 in the next release.

**Comment by andrcuns** [comment]
Sure, should be a pretty small PR :)

**Comment by butomo1989** [comment]
fixed in #3
